mne/forward/_compute_forward.py

```diff
@@ -814,110 +762,55 @@ def _prep_field_computation(rr, bem, fwd_data, n_jobs, verbose=None):
     #    fun (_bem_pot_or_field if not 'sphere'; otherwise _sph_pot_or_field)
     #    solutions (len 2 list; [ndarray, shape (n_MEG_sens, n BEM vertices),
     #                            ndarray, shape (n_EEG_sens, n BEM vertices)]
-    #    csolutions (compensation for solution)
-    fwd_data.update(dict(bem_rr=bem_rr, mri_Q=mri_Q, head_mri_t=head_mri_t,
-                         compensators=compensators, solutions=solutions,
-                         csolutions=csolutions, fun=fun,
-                         coils_list=coils_list, ccoils_list=ccoils_list))
+    fwd_data = dict(
+        bem_rr=bem_rr, mri_Q=mri_Q, head_mri_t=head_mri_t, fun=fun,
+        solutions=solutions)
```

Comment:
Note that now `fwd_data` is just stuff about transforms and BEM solutions, which I think is much cleaner than before!